components/NewsCard.vue

```diff
@@ -37,6 +42,8 @@ import { dateToHuman } from '@/utils'
 @Component
 export default class News extends Vue {
   @Prop() news!: NewsInterface | null
+  @Prop() index!: number
```

Comment:
You should pass what are the type that prop is expecting 
@Prop(Number) index!: number

Reply:
resolved